packages/icons/README.md

```diff
@@ -0,0 +1,21 @@
+# @vibe/icons
+
+This package includes all of the icons of monday.com's Vibe Design System
+
+## Installation
+
+To install, use the following command:
+
+```bash
+npm install @vibe/icons
+```
+
+## Usage
+
+```javascript
+import { Add } from "@vibe/icons";
```

Comment:
I'd choose something more classic than Add :) Might be confusing

Reply:
Changed it to "close", is that better?